components/03-modules/toolbar/_toolbar.scss

```diff
@@ -1,16 +1,36 @@
-$toolbar__offset               : $spacer !default;
-$toolbar__secondary-text-color : #a09fa6 !default;
-$toolbar__icon-color           : #a09fa6 !default;
-$toolbar__icon-color--active   : $black !default;
-$toolbar__select-border        : 1px solid $black !default;
-$toolbar__select-border-radius : 0 !default;
+$toolbar__dropdown-icon-area-border      : none !default;
+$toolbar__dropdown-icon-area-size        : 32px !default;
+$toolbar__dropdown-icon-color            : $gray !default;
+$toolbar__dropdown-icon-size             : 12px !default;
+$toolbar__dropdown-icon-area-margin      : 0 !default;
+$toolbar__dropdown-icon-area-offset-right: 0 !default;
+$toolbar__dropdown-icon-area-offset-top  : 2px !default;
+$toolbar__offset                         : $spacer !default;
+$toolbar__secondary-text-color           : $gray !default;
+$toolbar__icon-color                     : $gray-light !default;
+$toolbar__icon-color--active             : $black !default;
+$toolbar__mode-icon-area-size            : 48px !default;
+$toolbar__mode-icon-margin               : 0 auto !default;
+$toolbar__mode-icon-padding              : 0 !default;
+$toolbar__mode-icon-size                 : 18px !default;
+$toolbar__mode-margin-right              : $spacer !default;
+$toolbar__select-border                  : 1px solid $gray-light !default;
+$toolbar__select-border-radius           : $form-elements-radius !default;
+$toolbar__select-padding-right           : $spacer--medium !default;
```

Comment:
it's `toolbar__select-field-list-padding-right--single` 😄

Reply:
but don't know you want this so long variable

Reply:
not neccessary class name 1:1 to variable name.